styles/second-rectangle-container.scss

```diff
@@ -0,0 +1,11 @@
+@import "mixins.scss";
+
+.second-rectangle-container{
+  @include mediaScreenMaxWidth(720px){
+    position: static;
+  }
+
+  position: relative;
+  top: -13.75rem;
```

Comment:
In BEM you can't use external positioning in block
https://en.bem.info/methodology/css/#external-geometry-and-positioning